workflow.txt

```diff
@@ -8,3 +8,13 @@ git merge my-solution
 fork on github
 git checkout master
 git push origin my-solution
+
+fork on github
+git clone
+git checkout -b my-solution
```

Comment:
Great until this point
We need to add our files and make a commit before checking out master,
so the following lines need to happen just after `git checkout -b my-solution`
`git add .`
`git commit -m ""`
`git checkout master`
Now we pull any recent changes from the remote repo
`git pull upstream master`
merge those changes into our `my-solution` branch
`git merge my-solution`
and finally push the branch
`git push origin my-solution`